src/guc.c

```diff
@@ -65,6 +65,7 @@ bool ts_guc_enable_constraint_exclusion = true;
 bool ts_guc_enable_qual_propagation = true;
 bool ts_guc_enable_cagg_reorder_groupby = true;
 bool ts_guc_enable_now_constify = true;
+bool ts_guc_enable_foreign_key_propagation = true;
```

Comment:
I thought the decision was to not have a GUC and that the default would be to not propagate foreign keys. Or, is this something different?

Reply:
the guc is for support/debugging for foreign key propagation in general, it will not alter tiered storage behaviour